webflow-hacks/misc-hacks/7.ts

```diff
@@ -0,0 +1,18 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', function () {
+  /* get a reference to the link element
+  with a rel attribute of icon and store it in the variable link or 
+  if the link does not exist, create a link element
+  and store it in the variable link*/
+  // add selector constant
+  const LINK_SELECTOR = "[fs-hacks-element='link-icon']";
+  const link = document.querySelector<HTMLAnchorElement>(LINK_SELECTOR) || document.createElement('link');
```

Comment:
I think this should not be a custom selector querying a random anchor link on the page.
I believe the orignal script queries the `<link>` element that holds the favicon.